Content.Shared/Maps/ContentTileDefinition.cs

```diff
@@ -18,8 +21,8 @@ public sealed class ContentTileDefinition : IPrototype, IIndexedPrototype, ITile
         public ushort TileId { get; private set; }
         public string DisplayName { get; private set; }
         public string SpriteName { get; private set; }
+        public static Dictionary<string, List<string>> BaseTurfs { get; private set; } = new Dictionary<string, List<string>>();
```

Comment:
Why even use a static dictionary here? Just store the list on the tile definition directly.

Reply:
I thought the tile definitions weren't singletons at the time, and didn't want to produce a ton of lists. Error fixed.